### ISSUE TITLE
<button href="https://www.example.com"> not working

### ISSUE THREAD

**Issue by LeonidasJP** [Prezent/prezent-inky#1]
The ButtonFactory seems to be unable to parse a button and set the URL properly. Using v0.1.0 via Prezent/prezent-ink-bundle

In my Twig-template, I have:
```html
<button href="{{ url }}">
    {{ ('social.facebook'|trans }} - {{ url }}
</button>
```

With `url` being `https://www.example.com`, I expect the following output:
(inline styles stripped from output)
```html
<table class="button">
    <tr>
        <td>
            <table>
                <tr>
                    <td>
                        <a href="https://www.example.com">
                            Facebook - https://www.example.com
                        </a>
                    </td>
                </tr>
            </table>
        </td>
    </tr>
</table>
```

But Inky seems to end up with:
```html
<table class="button">
    <tr>
        <td>
            <table>
                <tr>
                    <td>
                        <a href="#">
                            Facebook - https://www.example.com
                        </a>
                    </td>
                </tr>
            </table>
        </td>
    </tr>
</table>
```

Why is the `<a>` still unlinked?

**Comment by sandermarechal** [comment]
This library knows nothing about Twig. You are probably converting Twig templates to HTML before feeding them to Inky. Can you post the actual HTML that you put into Inky?

The `href="#"` is a fallback for when the button has no `href` attribute.

Also, does the test suite pass successfully on your system? There is a test in the ButtonFactoryTest class that covers your exact situation.

**Comment by LeonidasJP** [comment]
From `Prezent\InkBundle\Mail\TwigFactory::renderHtmlPart()` this fragment is being dumped just before passing it to Inky:

```html
<button href="https://www.facebook.com/example">
    Facebook - https://www.facebook.com/example
</button>
```

But what Inky returns here is:
```html
<table class="button">
    <tr>
        <td>
            <table>
                <tr>
                    <td><a href="#">
                         Facebook - https://www.facebook.com/example
                     </a></td>
                </tr>
            </table>
        </td>
    </tr>
</table>
```

Locally, the test suite runs, without any errors.

**Comment by sandermarechal** [comment]
I cannot reproduce it. I tested the following code on various PHP versions and it always works. Can you provide a minimal test case for me?

```php
$inky = new Prezent\Inky\Inky();
echo $inky->parse('<button href="https://www.facebook.com/example">
    Facebook - https://www.facebook.com/example
</button>');
```

**Comment by LeonidasJP** [comment]
I have a dummy up and running [internally](http://leonh.equinoxe.dev9.intern.prezent.nl/temp.php)
Source can be found on Pastebin: https://pastebin.com/y6gbmd4x

**Comment by sandermarechal** [comment]
Fixed in 0.1.1
